### ISSUE TITLE
Request for Model Training Code

### ISSUE THREAD

**Issue by Moon1125666900** [jnnan/trumans_utils#5]
Hi,
I'm working on replicating the results from your paper and was hoping to access the training code for your models.
Would you consider open-sourcing the training code? It would greatly aid in the reproducibility of your work.
Thank you for considering my request.

**Comment by jnnan** [comment]
The training code will be released soon. Thank you for your interest in our project and for your patience.

**Comment by Moon1125666900** [comment]
Hello,

I have a few more questions regarding the project that I'd like to have clarified:

1. Could you please inform me when the action labels will be uploaded?
2. I am also curious about the process of generating the sequence of object poses.Regarding the output from the UNet part, it seems that I only see the pose sequences of the characters. Could you please confirm if this is the intended behavior or if there's something I might have missed?

Thank you for your time and assistance.

**Comment by jnnan** [comment]
The training code and the action labels are now released. Regarding the object poses, we attach the object to the character's hand after the grasping motion. We do not directly generate object poses. It is calculated from the character's pose instead.

**Comment by Moon1125666900** [comment]
According to the code you've made open-source, I set the batchsize to 512 in config_train_synhsi_trumans.yaml and used 4 GPUs. This is the result after training for 600 epochs on 4 RTX 3090s. Under these conditions, it seems that one epoch consists of about 1800 steps, totaling around 1000k steps of training. However, the results are significantly different from the visualization effects shown in your demo. There are issues such as people not following the route and experiencing sliding or teleportation. Could you please help me understand what might be causing this?
![4](https://github.com/user-attachments/assets/e384e8ea-d7c0-4e77-aaaf-5a46b76f6e5c)
![1](https://github.com/user-attachments/assets/84280352-784d-4c31-9601-b711e65f246e)

**Comment by ldfgood** [comment]
Pls how do you download the full dataset, every time I download the dataset with Google drive, there are always files missing. @Moon1125666900

**Comment by jnnan** [comment]
@Moon1125666900 Sorry for my late reply. Has this bug been fixed?

**Comment by Moon1125666900** [comment]
Yes,this bug has been fixed.


> @Moon1125666900 Sorry for my late reply. Has this bug been fixed?
